src/scss/layout/_header.scss

```diff
@@ -0,0 +1,105 @@
+.wrapper__haeder {
+  background-color: $color-border-header;
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 6px solid $color-border-header;
+  position: fixed;
+  z-index: 1000;
+
+  @media (max-width: 767px) {
+    position: static;
+  }
+  @media (max-width: 376px) {
+    position: static;
+  }
+}
```

Comment:
Code style here. You add one empty line before @media, but no empty line before the second @media.